web/Services/LandlordService.cs

```diff
@@ -77,4 +78,19 @@ await using (var transaction = await _dbContext.Database.BeginTransactionAsync(I
 
         return ILandlordService.LandlordRegistrationResult.Success;
     }
+
+    public async Task ApproveLandlord(int landlordId,  BricksAndHeartsUser user)
+    {
+        var landlord = _dbContext.Landlords.Single(l =>l.Id == landlordId);
+        landlord.CharterApproved = true;
+        landlord.ApprovalTime = DateTime.Now;
+        landlord.ApprovalAdminId = user.Id;
+        await _dbContext.SaveChangesAsync();
+    }
+    
+    public List<PropertyDbModel> GetListOfProperties(int landlordId)
+    {
+        return _dbContext.Properties
+            .Where(p => p.LandlordId == landlordId).ToList();
+    }

```

Comment:
Is this intentional? Possibly from a merge

Reply:
I just asked Max, apparently it was replaced by a method in PropertyServices - the merge must have accidentally kept both versions.

Reply:
I've deleted it now, at least from this branch.